roles/migrationcontroller/defaults/main.yml

```diff
@@ -62,6 +62,10 @@ velero_plugin_version: "{{ snapshot_tag | default(lookup( 'env', 'VELERO_PLUGIN_
 velero_restic_restore_helper_image: "{{ registry }}/{{ project }}/{{ velero_restic_restore_helper_repo }}"
 velero_restic_restore_helper_repo: "{{ lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_REPO') }}"
 velero_restic_restore_helper_version: "{{ snapshot_tag | default(lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_TAG')) }}"
+velero_exclude_resources:
+  - clusterserviceversions
+  - subscriptions
+  - templateinstances
```

Comment:
We could add this to the MigrationController CR examples too. I'm a fan of keeping those relatively sparse and documentation leading the way on explaining how to override the defaults. After awhile it will be hard to navigate all the options. On the other hand I think we've A) seen the documentation is lacking, and B) it's not really clear for folks to lookup what can be changed (i.e locating the defaults/main.yml).

Reply:
@jwmatthews do you have an opinion?